### PR TITLE
Move embedded code example in articles to code repo

### PR DESCRIPTION
**Pull request by dksimpson**
## Summary

Related to .NET refresh work for task 1409972: https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1409972/ and 
1409954: https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1409954/
See PRs #https://github.com/dotnet/docs/pull/11447, #https://github.com/dotnet/docs/pull/11461

As a best practice, move the C# and VB complete code example from the following articles to the samples repo:
https://docs.microsoft.com/en-us/dotnet/framework/network-programming/how-to-request-data-using-the-webrequest-class
https://docs.microsoft.com/en-us/dotnet/framework/network-programming/how-to-send-data-using-the-webrequest-class

### CODE REVIEW

File: snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            // Get the request stream.  
+            Stream dataStream = request.GetRequestStream ();  
+            // Write the data to the request stream.  
+            dataStream.Write (byteArray, 0, byteArray.Length);  
+            // Close the Stream object.  
+            dataStream.Close ();  
+            // Get the response.  
+            WebResponse response = request.GetResponse ();  
+            // Display the status.  
+            Console.WriteLine (((HttpWebResponse)response).StatusDescription);
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs]:
```suggestion
            Console.WriteLine(((HttpWebResponse)response).StatusDescription);  
```

File: snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            // Write the data to the request stream.  
+            dataStream.Write (byteArray, 0, byteArray.Length);  
+            // Close the Stream object.  
+            dataStream.Close ();  
+            // Get the response.  
+            WebResponse response = request.GetResponse ();  
+            // Display the status.  
+            Console.WriteLine (((HttpWebResponse)response).StatusDescription);  
+            // Get the stream containing content returned by the server.  
+            dataStream = response.GetResponseStream ();
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs]:
```suggestion
            dataStream = response.GetResponseStream();  
```

File: snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+            // Close the Stream object.  
+            dataStream.Close ();  
+            // Get the response.  
+            WebResponse response = request.GetResponse ();  
+            // Display the status.  
+            Console.WriteLine (((HttpWebResponse)response).StatusDescription);  
+            // Get the stream containing content returned by the server.  
+            dataStream = response.GetResponseStream ();  
+            // Open the stream using a StreamReader for easy access.  
+            StreamReader reader = new StreamReader (dataStream);
```
Comment (rpetrusha) [inline snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs]: this should be wrapped in a `using` statement.

File: snippets/visualbasic/VS_Snippets_Network/RequestDataUsingWebRequest/vb/WebRequestGetExample.vb [hunk 4]
```diff
@@ -0,0 +1,31 @@
+Imports System
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Network/RequestDataUsingWebRequest/vb/WebRequestGetExample.vb]: System never needs to be imported.

File: snippets/visualbasic/VS_Snippets_Network/RequestDataUsingWebRequest/vb/WebRequestGetExample.vb [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            ' If required by the server, set the credentials.  
+            request.Credentials = CredentialCache.DefaultCredentials  
+            ' Get the response.  
+            Dim response As WebResponse = request.GetResponse()  
+            ' Display the status.  
+            Console.WriteLine(CType(response,HttpWebResponse).StatusDescription)  
+            ' Get the stream containing content returned by the server.  
+            Dim dataStream As Stream = response.GetResponseStream()  
+            ' Open the stream using a StreamReader for easy access.  
+            Dim reader As New StreamReader(dataStream)
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Network/RequestDataUsingWebRequest/vb/WebRequestGetExample.vb]: This should be wrapped in a using statement and the call to reader.Close removed.

File: snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb [hunk 6]
```diff
@@ -0,0 +1,44 @@
+Imports System
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb]: remove this line.

File: snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb [hunk 7]
```diff
@@ -0,0 +1,44 @@
+Imports System  
+Imports System.IO  
+Imports System.Net  
+Imports System.Text
```
Comment (rpetrusha) [inline snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb]: Here again, I don't think any System.Text types are used.

File: snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs [hunk 8]
```diff
@@ -0,0 +1,47 @@
+using System;  
+using System.IO;  
+using System.Net;  
+using System.Text;
```
[the remark this answers is not in the document]
Comment (dksimpson) [reply snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs]: According to Visual Studio, System.Text is used by Encoding, in byte[] byteArray = Encoding.UTF8.GetBytes(postData);

File: snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb (hunk 7 above)
Comment (dksimpson) [reply snippets/visualbasic/VS_Snippets_Network/SendDataUsingWebRequest/vb/WebRequestPostExample.vb]: According to VS, Encoding needs System.Text in Dim byteArray As Byte() = Encoding.UTF8.GetBytes(postData)

**Comment by dksimpson** [comment]
Hi @rpetrusha, thanks for your review comments - I've made your suggested changes. Please confirm that my using statement blocks have the correct syntax.

File: snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs (hunk 8 above)
[the remark this answers is not in the document]
Comment (rpetrusha) [reply snippets/csharp/VS_Snippets_Network/SendDataUsingWebRequest/cs/WebRequestPostExample.cs]: Sorry. I completely missed the presence of Encoding.

**Pull request merged**